### PR TITLE
New adjustments in connectivity

### PR DESCRIPTION
**Pull request by cazapatamar**

### CODE REVIEW

**Review by Trjegul84 — CHANGES_REQUESTED** [review]
I've highlight in red some values that are not completely visible, I think the numbers are to long to be there and not placed well. Are those necessary at the left of the bars?

Cifras-NoVisibles

**Comment by cazapatamar** [comment]
> I've highlight in red some values that are not completely visible, I think the numbers are to long to be there and not placed well. Are those necessary at the left of the bars?
> 
> Cifras-NoVisibles

It seems to be a problem with nivo upgrade you did in the other PR. Try to delete node_modules, install dependencies again been on this branch and test again. In my local env i don't see labels at Y-axis as it should be.

image

if indeed it is issue related to nivo upgrade, please solve it in the PR #459

**Comment by Trjegul84** [comment]
Ok, thanks, I will check again.

**Comment by erikasv** [comment]
@cazapatamar are you suggesting we should merge this branch first?, because if that problem happens because of the nivo upgrade it equally has to be solved

**Comment by cazapatamar** [comment]
> @cazapatamar are you suggesting we should merge this branch first?, because if that problem happens because of the nivo upgrade it equally has to be solved

I mean, first solve the issue but in the other PR where we are upgrading nivo and not here. What do you think? @erikasv

**Comment by erikasv** [comment]
> I mean, first solve the issue but in the other PR where we are upgrading nivo and not here. What do you think? @erikasv

Yeah agree, I just saw your comment in the other PR. I just was worried the problem wouldn't be solved.

**Comment by cazapatamar** [comment]
> > I mean, first solve the issue but in the other PR where we are upgrading nivo and not here. What do you think? @erikasv
> 
> Yeah agree, I just saw your comment in the other PR. I just was worried the problem wouldn't be solved.

@erikasv After checking with @Trjegul84 we realize the issue is not associated to nivo upgrade but to an error inside the component as i was setting the axisLeft prop instead of being null. It was already fixed in the last commit.

File: src/components/charts/GraphLoader.jsx [hunk 1]
```diff
@@ -92,7 +92,6 @@ const GraphLoader = (props) => {
           onClickHandler={onClickGraphHandler}
           selectedIndexValue={selectedIndexValue}
           labelX={labelX}
-          labelY={labelY}
```
Comment (erikasv) [inline src/components/charts/GraphLoader.jsx:95]: Wouldn't this affect the Loss and persistence graph?

File: src/components/charts/MultiSmallSingleBarGraph.jsx [hunk 2]
```diff
@@ -20,7 +20,6 @@ class MultiSmallSingleBarGraph extends React.Component {
       units,
       onClickHandler,
       labelX,
-      labelY,
```
Comment (erikasv) [inline src/components/charts/MultiSmallSingleBarGraph.jsx:23]: Same that previous comment, this wouldn't affect loss and persistence section?

File: src/components/charts/GraphLoader.jsx (hunk 1 above)
Comment (cazapatamar) [reply src/components/charts/GraphLoader.jsx:95]: No, the component is different for Loss and persistence graph whose name is MultiSmallBarStackGraph

File: src/components/charts/MultiSmallSingleBarGraph.jsx (hunk 2 above)
Comment (cazapatamar) [reply src/components/charts/MultiSmallSingleBarGraph.jsx:23]: Same last comment

**Review by Trjegul84 — APPROVED** [review]

**Review by erikasv — APPROVED** [review]

**Pull request merged**